### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd`. No need to set `AFL_SKIP_CPUFREQ`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sdr-server-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: You may omit image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**